scripts/CETB_read_functions.py

```diff
@@ -228,86 +320,40 @@ def coords(datadir, prefix, lat_start, lat_end, lon_start, lon_end):
-    return DAV_abs
-
-# find the offset to be used for locating the grid row/col
-# from lat/lon coordinates (GLaIL)
-def find_GLaIL_cube_offset(cubeType=None, verbose=False):
-    if not cubeType:
-        cubeType = '36V-SIR'
-
-    cubeFile = "%s%s%s%s" % (
-        "/home/mij216/Desktop/data3/cetb/cubes/AQUA_AMSRE/GLaIL/",
-        "CETB.cubefile.GLaIL.AQUA_AMSRE-",
-        cubeType,
-        "-RSS-v1.3.2003.TB.nc")
-
-    f = Dataset(cubeFile, "r", "NETCDF4")
-    lats = f.variables['latitude'][:]
-    lons = f.variables['longitude'][:]
-    baseGPD = f.variables['crs'].long_name
-    f.close()
-
-    # find and return the baseGPD (row, col) offset for cubeUL(0, 0) location
-    grid = Ease2Transform(baseGPD)
-    row_offset, col_offset = grid.geographic_to_grid(lats[0,0], lons[0,0])
-
-    if verbose:
-        print("%10s offsets for cubeType=%s" % (baseGPD, cubeType))
-        print("(Add these offsets to cube (row, col)"+
-              "to get row, col in full hemisphere)")
-        print("offset row = %f, offset col = %f" % (row_offset, col_offset))
-
-    return row_offset, col_offset
-
-# offset for Western US cubes
-def find_WesternUS_cube_offset(cubeType=None, verbose=False):
-    if not cubeType:
-        cubeType = '36V-SIR'
 
-    cubeFile = "%s%s%s%s" % (
-        "/home/mij216/Desktop/data3/cetb/cubes/AQUA_AMSRE/WesternUS/",
-        "CETB.cubefile.WesternUS.AQUA_AMSRE-",
-        cubeType,
-        "-RSS-v1.3.2003.TB.nc")
+    # function takes the CETB_data that was read in read_Tb() and returns the absolute value of the DAV         
+    DAV=np.diff(CETB_data,n=1,axis=0)
+    DAV_abs=np.absolute(DAV)    
+    DAV_abs=np.insert(DAV_abs, [0],[0], axis=0)         # insert a 0 at beginning of array so same length as CETB_data for plotting together
 
-    f = Dataset(cubeFile, "r", "NETCDF4")
-    lats = f.variables['latitude'][:]
-    lons = f.variables['longitude'][:]
-    baseGPD = f.variables['crs'].long_name
-    f.close()
-    # find and return the baseGPD (row, col) offset for cubeUL(0, 0) location
-    grid = Ease2Transform(baseGPD)
-    row_offset, col_offset = grid.geographic_to_grid(lats[0,0], lons[0,0])
+    return DAV_abs
 
-    if verbose:
-        print("%10s offsets for cubeType=%s" % (baseGPD, cubeType))
-        print("(Add these offsets to cube (row, col) "+
-              "to get row, col in full hemisphere)")
-        print("offset row = %f, offset col = %f" % (row_offset, col_offset))
+# find the offset to be used for locating the grid row/col from lat/lon coordinates
+# for the requested subsetName
+# subsetNames are, for e.g. "GLaIL" or "WesternCA" etc.
+def find_cube_offset(subsetName, cubeDir=None, cubeType=None, verbose=False):
 
-    return row_offset, col_offset
+    # when no cubeDir is specified, assume it's in a regular
+    # location on fringe        
+    if not cubeDir:
+        cubeDir = "%s%s" % (
+            "/home/mij216/Desktop/data3/cetb/cubes/AQUA_AMSRE/",
```

Comment:
on fringe the default location
/mnt/data3/cetb/cubes/ 
followed by the sensor name